components/Tooltip/TimerTooltip.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { TimerTooltip, ToolTipWrapper } from './TooltipStyles';
+import CountUpTimer from '../../sections/futures/CountUpTimer';
+
+interface ToolTipProps {
+	startTimeDate: Date | undefined;
+	children?: React.ReactNode;
+	preset?: string;
+	top?: string;
+	bottom?: string;
+	left?: string;
+	right?: string;
+}
+
+const GeneralTooltip = (props: ToolTipProps) => {
```

Comment:
Just curious about why you named this component as `GeneralTooltip` and then put `TimerTooltip` into it.